### ISSUE TITLE
Scalar bar color cannot be changed in trame local view

### ISSUE THREAD

**Issue by Jimmy-KL** [pyvista/pyvista#3771]
### Describe the bug, what's wrong, and what you expected.

@banesullivan I'm using branch _feat/trame_ and I found that if I'm using PyVistaLocalView, the color of the scalar bar cannot be changed. I set it to white, but it's still black.
But for PyvistaRemoteView, the _scalar bar arg color_ is working. I think the problem is that the scalar bar widget for the local view is different from the remote view.

### Steps to reproduce the bug.

Here is the related code.
`plotter.add_mesh(mesh, name='model', scalar_bar_args={'color':'white'})`
It's working for remote view.

### System Information

```shell
OS : Windows
            CPU(s) : 16
           Machine : AMD64
      Architecture : 64bit
       Environment : Python
       GPU Details : error

  Python 3.9.7 (tags/v3.9.7:1016ef3, Aug 30 2021, 20:19:38) [MSC v.1929 64 bit
  (AMD64)]

           pyvista : 0.38.dev0
               vtk : 9.1.0
             numpy : 1.23.3
           imageio : 2.22.0
            scooby : 0.5.12
             pooch : v1.6.0
        matplotlib : 3.6.0
             trame : Version unknown
         trame_vtk : Version unknown
```

### Screenshots

Local View - the color of the scalar bar is always black.
![image](https://user-images.githubusercontent.com/114913030/210248567-7136faf3-dc93-4ec7-bc75-72531945c012.png)

Remote View - the color of the scalar bar can be controlled.
![image](https://user-images.githubusercontent.com/114913030/210248714-cc87be96-c9ec-4b26-9181-3234cf97b455.png)

**Comment by banesullivan** [comment]
We actually fixed this upstream in trame not too long ago in https://github.com/Kitware/trame-vtk/pull/5

Can you upgrade `trame-vtk` to the latest version? 

Side note: I see PyVista's report is having trouble getting the version of trame so I'll need to fix that

**Comment by banesullivan** [comment]
Those changes in trame are patches in that they override the serializer in VTK Python. I added these same changes to VTK which should make it to the 9.2.2 release: https://gitlab.kitware.com/vtk/vtk/-/merge_requests/9766

**Comment by Jimmy-KL** [comment]
I upgraded trame-vtk  to the latest version, check below:

Name: trame-vtk
Version: 2.0.12
Summary: VTK widgets for trame
Home-page: 
Author: Kitware Inc.
Author-email: 
License: BSD License
Location: d:\offlinework\trame-tutorial\.venv\lib\site-packages
Requires: trame-client
Required-by: trame

But the problem is still there - I cannot change the color of the scalar bar for local view, but it's working for remote view.
BTW, the default style and location of the scalar bar for the local and remote view is different too. So I think the scalar bar in the local view is not the scalar bar in Pyvista.

**Comment by banesullivan** [comment]
Okay, thanks for reporting. I'll dig into this

**Comment by Jimmy-KL** [comment]
I found that the bounds box cannot be modified in the local view, either. It seems some widgets or elements in the local view cannot be modified. I think it's an upstream bug of trame-vtk.

**Comment by Jimmy-KL** [comment]
OK, thanks, I will take a look.

**Comment by PBrockmann** [comment]
It seems not fixed.

```pip install --upgrade trame-vtk
Requirement already satisfied: trame-vtk in /opt/anaconda3-2022.10/envs/myenv/lib/python3.10/site-packages (2.0.17)
Requirement already satisfied: trame-client in /opt/anaconda3-2022.10/envs/myenv/lib/python3.10/site-packages (from trame-vtk) (2.6.1)
```

**Comment by banesullivan** [comment]
@PBrockmann, please share a pyvista.Report()

**Comment by PBrockmann** [comment]
![image](https://user-images.githubusercontent.com/5402758/218885351-ee16320a-2fda-4ea7-b7c8-d3faf464220a.png)

**Comment by banesullivan** [comment]
Can you share screenshots of it not working?

**Comment by PBrockmann** [comment]
When 
```
pl.show(jupyter_backend='trame')
```

![image](https://user-images.githubusercontent.com/5402758/219030754-5115be24-00fc-49ac-8b13-3847a393151d.png)

When 
```
pl.show(jupyter_backend='client')
```

![image](https://user-images.githubusercontent.com/5402758/219031045-9ef32c67-2371-426c-a094-07b0c9de8695.png)

without changing anything else.

**Comment by banesullivan** [comment]
These are different problems than what this issue was created for.

> Note the quality that is much better with the backend "client".

client/server modes are fundamentally different. The server-mode streams screenshots to the browser where as the `client` mode renders natively in the browser and this is more "crisp". You can increase the "crispness" of the `server` mode with some global settings:

```py
# Scale the server-rendered image
pv.global_theme.trame.interactive_ratio = 2
pv.global_theme.trame.still_ratio = 2
```


> I cannot control the font of the scalar bar, neither its format.

The fonts do not synchronize (at this time). I opened an issue to track this here: https://github.com/Kitware/trame-vtk/issues/28

> BTW, is there a way to have a reduced menu with only

Please open a separate issue as a feature request for this to be considered. Currently, the UI is collapsable.
